### ISSUE TITLE
Customizable message for notification

### ISSUE THREAD

**Issue by devopshelper4u** [louislam/uptime-kuma#2312]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification, UI Feature

### 🔖 Feature description

Currently when a monitor goes up/down, there is a standard notification message like Down/UP and status code.

Instead can we have some customized message there where user can provide which link to display, link to route back to kuma monitor page, etc.

Also ability to customize icon, etc.

### ✔️ Solution

Basically ability to modify messages as well in alert msg.

**Comment by CommanderStorm** [comment]
Just noticed this is a duplicate of https://github.com/louislam/uptime-kuma/issues/646
We now support this via smtp, telegram and webhooks
